tests/test_integration.py

```diff
@@ -160,6 +161,23 @@ def check_vm_absent(self, hv_name=None):
                     )
 
 
+class HypervisorCpuThresholdsTest(IGVMTest):
+    """Test that all hardware_models of all hypervisors have a cpu threshold
+    value defined in HYPERVISOR_CPU_THRESHOLDS dict"""
+
+    def test_hypervisor_cpu_thresholds(self):
+        hardware_models = set([
+            x['hardware_model'] for x in Query({
+                'servertype': 'hypervisor',
+                'project': 'ndco',
+                'environment': 'production',
```

Comment:
All HVs should be checked, testing ones too.

Reply:
I extended it to production and testing

Reply:
just remove the `environment` filter altogether